operator/pkg/module/sync/runner_impl.go

```diff
@@ -169,3 +171,36 @@ func stateFromUnstructured(obj *unstructured.Unstructured) v1alpha1.State {
 	}
 	return v1alpha1.StateError
 }
+
+func (r *runnerImpl) UpdateModuleInfos(ctx context.Context, kyma *v1alpha1.Kyma) bool {
```

Comment:
I think this can become unexported
```suggestion
func (r *runnerImpl) updateModuleInfos(ctx context.Context, kyma *v1alpha1.Kyma) bool {
```

Reply:
done